mev_inspect/inspect_block.py

```diff
@@ -209,6 +219,11 @@ async def inspect_many_blocks(
     )
     write_punk_snipes(inspect_db_session, punk_snipes)
 
+    delete_nft_trades_for_blocks(
+        inspect_db_session, after_block_number, before_block_number
+    )
+    write_nft_trades(inspect_db_session, nft_trades)
```

Comment:
a bug (that exists in main as well, PR coming momentarily) is this should be `all_nft_trades)